hieradata/org/lsst/role/ccs-mcm.yaml

```diff
@@ -3,6 +3,7 @@ classes:
   - "profile::core::common"
   - "profile::ccs::common"
   - "profile::ccs::tomcat"
+  - "profile::ccs::graphical"
```

Comment:
FYI - I usually `sort` the inclusion list but the linter doesn't care.

Reply:
Noted.